### DISCUSSION TITLE
No puedo entrar con el correo y contraseña de innovacion que me proporcionaron

### DISCUSSION THREAD

**Discussion by FelipeBENIITEZ** [Launch-X-Latam/MisionFrontEnd#449] [Team Posts]
al ingresar el correo y contraseña de innovacion me dice que la contraseña ya expiro

**Comment by gersondlarosa** [comment]
Puedes mandar un mensaje al Instagram de innovacion y ellos te pueden apora con tu cuenta

**Comment by FelipeBENIITEZ** [comment]
ya lo hice y no he recibido respuesta
